app/scripts/controllers/membership.js

```diff
@@ -88,16 +91,16 @@ angular
           .then(function() {
             resetForm();
             refreshRoleBindingList();
-            showAlert('rolebindingCreate', 'success', messages.update.subject.success({
+            showAlert('rolebindingCreate', 'success', escaped.update.subject.success({
               roleName: role.metadata.name,
               subjectName: _.escape(newSubject.name)
```

Comment:
Any template that will displayed a plain text message in an alert should use `<%=`.

Any template that will display HTML content in a dialog should use `<%-` to escape the HTML. The only time you need to use this if you use `ng-bind-html` like in the dialog.

You shouldn't need to use `_.escape` anywhere if you use `<%-` for the HTML messages.